app/models/presenter/auction.rb

```diff
@@ -269,7 +88,7 @@ def markdown
                                             fenced_code_blocks: true,
                                             lax_spacing: true)
     end
-
+    
     def model
       __getobj__
     end
```

Comment:
It looks like this class shrank a lot. Wondering what criteria was for moving code into the policy object?